vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "vite";
+import path from "path";
+
+export default defineConfig({
+  resolve: {
+    alias: [
+      { find: "@", replacement: path.resolve(__dirname, "src") },
+      { find: "@view", replacement: path.resolve(__dirname, "src/view") },
+      { find: "@model", replacement: path.resolve(__dirname, "src/model") },
+      { find: "@util", replacement: path.resolve(__dirname, "src/util") },
+      { find: "@init", replacement: path.resolve(__dirname, "src/init") },
+      { find: "@handler", replacement: path.resolve(__dirname, "src/handler") },
```

Comment:
(마이너)

```suggestion
      { find: "@", replacement: path.resolve(__dirname, "src") },
```

그냥 이렇게 써도 될거 같아요!
[vite alias docs](https://ko.vitejs.dev/config/shared-options.html#resolve-alias)